docs/source/ecf.rst

```diff
@@ -1274,7 +1274,8 @@ This file describes the transit/eclipse and systematics parameters and their pri
 Available fitting parameters are:
 
    - Transit and Eclipse Depth Parameters
-      - ``rp`` or ``rprs`` - planet-to-star radius ratio, for the transit models.
+      - ``rp`` or ``rprs`` - planet-to-star radius ratio, for most transit models.
+      - ``a0`` - planet-to-star radius ratio for Harmonica.  Add  ``a1`` to fit different morning/evening limbs.  Add ``b1`` to fit different north/south limbs.  Higher-order harmonics (up to ``n=3``) are also available.  We recommend starting with ``a0`` and ``a1`` as free parameters. See the `Harmonica documentation <https://harmonica.readthedocs.io/en/latest/views/transmission_strings.html#>`__ for a detailed description of transmission strings and the Fourier series used to parameterize them.
```

Comment:
For simplicity, would it be better to link `rp` to `a0` within the `PlanetParams` class so that users can still use `rp` instead of `a0`? I'm imagining something similar to how our `rp` parameter is tied to our `rprs` parameter. Or is `a0` sufficiently different from `rp` that we don't want to confuse the two?

Reply:
Great idea! The two are interchangeable.